app/src/main/java/com/example/calculator/Validator.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.calculator
+
+import java.math.BigDecimal
+
+class Validator {
```

Comment:
является util классом, можно через singleton 
```suggestion
object Validator {
```